### PR TITLE
Change data source for South Australia battery

### PR DESCRIPTION
**Pull request by systemcatch**
Use nemlog api rather than global-roam.

ref #869

### CODE REVIEW

File: parsers/lib/AU_battery.py [hunk 1]
```diff
@@ -1,31 +1,40 @@
...
-    json_content = req.json()
-    dicts = json_content["seriesCollection"]
-    discharge_id = (item for item in dicts if item["id"] == "41").next()
-    storage_id = (item for item in dicts if item["id"] == "57").next()
+    today = arrow.now('Australia/Adelaide')
+    current = today.format('YYYYMMDD')
+    old = today.shift(days=-2).format('YYYYMMDD')
+    nemlog_url = 'http://nemlog.com.au/api/unit/HPRL1/{}/{}/json'.format(old, current)
+
+    req = requests.get(nemlog_url)
```
Comment (corradio) [inline parsers/lib/AU_battery.py]: is it possible to pass a session object in order to reuse existing one?

File: parsers/lib/AU_battery.py [hunk 2]
```diff
@@ -1,31 +1,40 @@
...
 
-    #Verify set to false to prevent SSL error due to domain mismatch.
-    req = requests.get(base_url, verify = False)
-    json_content = req.json()
-    dicts = json_content["seriesCollection"]
-    discharge_id = (item for item in dicts if item["id"] == "41").next()
-    storage_id = (item for item in dicts if item["id"] == "57").next()
+    today = arrow.now('Australia/Adelaide')
+    current = today.format('YYYYMMDD')
+    old = today.shift(days=-2).format('YYYYMMDD')
```
Comment (corradio) [inline parsers/lib/AU_battery.py]: why two days back?

Comment (systemcatch) [reply parsers/lib/AU_battery.py]: The default behavior of the api with no dates is to return data up to 2 days back.  I've kept that for the request but tbh as we only use the last value it doesn't make much difference.

**Comment by corradio** [comment]
btw did you test using the mockserver?

**Comment by systemcatch** [comment]
Yes it's working fine though the 0.2 MW of charging isn't shown on the breakdown (assuming rounded down).

**Comment by corradio** [comment]
ok great. Feel free to commit the update state.json so we have it as reference (and for others)

**Pull request merged**